client/src/index.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom'
```

Comment:
If you wanted to rename BrowserRouter to use a smaller name you can do: 
`import {BrowserRouter as Router} from 'react-router-dom'`

<Router></Router> would work the same as the <BrowserRouter></BrowserRouter> just less typing